launchable/commands/subset.py

```diff
@@ -458,6 +470,17 @@ def run(self):
 
                     res = client.request("post", "subset", timeout=timeout, payload=payload, compress=True)
 
+                    # The status code 422 is returned when validation error of the test mapping file occurs.
+                    if res.status_code == 422:
+                        msg = "Error: {}".format(res.json().get("reason"))
+                        tracking_client.send_error_event(
+                            event_name=Tracking.ErrorEvent.INTERNAL_CLI_ERROR,
```

Comment:
Is it an internal error? I think the error is caused by the user. WDYT??

Reply:
[7389429](https://github.com/launchableinc/cli/pull/622/commits/7389429c6b9057478972edd1d7bdf7312c6f7a0f)